### PR TITLE
Add io-hog chaos scenario

### PR DESCRIPTION
**Pull request by chaitanyaenr**

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
Depends on https://github.com/redhat-chaos/krkn-hub/pull/140. Thanks.

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414 periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414 periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414 periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

File: ci-operator/step-registry/redhat-chaos/hog-scenarios/io/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+approvers:
```
Comment (paigerube14) [inline ci-operator/step-registry/redhat-chaos/hog-scenarios/io/OWNERS:1]: This file should reference the OWNERS file in the main redhat-chaos folder, this way we only need to update one file for the whole repo and it'll stay up to date

you can create a symbolic link using
ln -s ./OWNERS ./<folder_name>

**Pull request merged**